client/src/App.css

```diff
@@ -0,0 +1,27 @@
+.remove-btn {
+    margin-right: 2.5rem;
+}
+
+.add-btn {
+    position: absolute;
+    right: 100px;
+    top: 140px;
+}
+
+.fade-enter {
+    opacity: 0.01;
+}
+
+.fade-enter-active {
+    opacity: 1;
+    transition: opacity 1000ms ease-in;
+}
+
+.fade-exit {
+    opacity: 1;
+}
+
+.fade-exit-active {
+    opacity: 0.01;
+    transition: opacity 1000ms ease-in;
+}
```

Comment:
You can delete the contents of this file. We are not using it.

Reply:
Can I actually keep this one? Because otherwise the Add Button will remain on the left.

I could put it in the style but I already deleted keys.js.

Reply:
Sure.